docs/configuration.rst

```diff
@@ -171,6 +171,13 @@ SMTP
 
 Note that the SMTP server shipped with Tutor by default does not implement TLS. With external servers, only one of SSL or TLS should be enabled, at most.
 
+Forum
+*****
+
+- ``RUN_FORUM`` (default: ``true``)
+- ``FORUM_HOST`` (default: ``forum``)
+- ``FORUM_MONGODB_DATABASE`` (default: ``cs_comments_service``)
```

Comment:
Sorry to nitpick @ChrisChV, but could you add "" quotes around the values for `FORUM_HOST` and `FORUM_MONGODB_DATABASE`? This is to indicate that the type of the setting value is a string, as for other items in the docs. Thus:

```
- ``RUN_FORUM`` (default: ``true``)
- ``FORUM_HOST`` (default: ``"forum"``)
- ``FORUM_MONGODB_DATABASE`` (default: ``"cs_comments_service"``)
```